tools/SampleAnalyzer/Interfaces/fastjet/ClusterAlgoFastJet.cpp

```diff
@@ -163,37 +78,65 @@ MAbool ClusterAlgoFastJet::Execute(SampleFormat& mySample, EventFormat& myEvent,
   MAfloat64 & THT = myEvent.rec()->THT();
   MAfloat64 & Meff= myEvent.rec()->Meff();
 
+
   // Storing
-  for (MAuint32 i=0;i<jets.size();i++)
+  for (MAuint32 ijet=0;ijet<jets.size();ijet++)
```

Comment:
@BFuks I'm thinking about moving the entire tagging module into this loop. We can merge truth and SFS level tagging together and trigger SFS if there is an old version of tagging commands in place so that the python interface would be unchanged. Do you agree?